src/runtime_src/core/pcie/emulation/common_em/unix_socket.cxx

```diff
@@ -19,25 +19,22 @@
 
 #include "unix_socket.h"
 
-unix_socket::unix_socket()
+unix_socket::unix_socket(std::string sock_id)
```

Comment:
const std::string& unless the string is moved, which is doesn't appear to be.

Reply:
Yes. This is fixed